pom.xml

```diff
@@ -109,6 +109,11 @@
             <version>${automation.framework.utils.version}</version>
             <!--<scope>compile</scope>-->
         </dependency>
+        <dependency>
+            <groupId>org.wso2.org.apache.commons</groupId>
+            <artifactId>commons-vfs2</artifactId>
+            <version>2.2-wso2v4</version>
```

Comment:
Better to move this version to a property.

Reply:
done